src/entities/tag.rs

```diff
@@ -7,5 +7,6 @@ pub struct Tag {
     pub name: String,
     pub url: String,
     pub history: Option<Vec<History>>,
+    #[serde(default)]
```

Comment:
If this field is an optional field, it is better `Option<bool>` I think. If my understanding is correct, `serde(default)` injects the default value. That means when the tag does not contain `following` field, it always returns `false`.

Reply:
Well, if it is an optional the assumption would be that it is `false` when `None`, wouldn't it? Having a default `false` makes usage of the entity a bit more pleasant:

``` rs
// a:
let is_following = entity.following.unwrap_or_default();
// b:
if let Some(is_following) = entity.is_following {
  if is_following {
  }
}

vs:
if is_following {
}
```

I fail to see a case where a `Optional<bool>` offers a benefit. I think it would make sense if there are three states: User is following, user is not following and user hasn't set this flag yet.

Reply:
Hmm, megalodon is a client library, so I think the response from the server should be returned as accurately as possible. I don't understand why this library returns `false` when the server doesn't return `following`.

Reply:
Point taken, I'll change it :)